lib/main_common.pm

```diff
@@ -1696,7 +1697,11 @@ sub load_create_hdd_tests {
     loadtest 'console/hostname'       unless is_bridged_networking;
     loadtest 'console/force_cron_run' unless is_jeos;
     # Remove repos pointing to download.opensuse.org and add snaphot repo from o3
-    replace_opensuse_repos_tests if is_repo_replacement_required;
+    if (is_repo_replacement_required) {
+        loadtest 'console/consoletest_setup';
+        replace_opensuse_repos_tests;
+        loadtest 'console/consoletest_finish';
```

Comment:
It was not the idea of "consoletest_setup" and "consoletest_finish" to wrap just one module. I doubt this is a good idea. Please just include the necessary parts from consoletest_setup within `replace_opensuse_repos_tests`. Certainly most parts are *not* necessary. And especially consoletest_finish I think should not be needed at all here.

Reply:
consoletest_finish reverts some parts we do in consoletest_setup. Like unmasking packagekit service. I'll check what we can do better here.